### ISSUE TITLE
Read less data in the Cell view?

### ISSUE THREAD

**Issue by severo** [hyparam/hyperparam-cli#8]
Maybe the column could be specified when fetching the data here:

https://github.com/hyparam/hyperparam-cli/blob/ca429b826bc33a8aa6173a8a4a9a4ba58cc16008/src/components/Cell.tsx#L43-L44

It would avoid downloading the whole row.

(maybe this optimization is not important, since ParquetView downloads multiple rows with all the cells and it's already fast)

**Comment by platypii** [comment]
Good point. The `DataFrame` interface will probably need the option to request a subset of the columns in order to make this more efficient.

**Comment by platypii** [comment]
Fixed by DataFrame V2 🎉
